docs/guides/tfhers.md

```diff
@@ -0,0 +1,242 @@
+# TFHE-rs Compatibility
+
+This document explains how to combine Concrete and [TFHE-rs](https://github.com/zama-ai/tfhe-rs) computations together. This feature targets two main use cases.
```

Comment:
I would extend this a bit: explain what you mean by "combine" or "compatibility". The idea is to be able to have ciphertexts of one kind and convert them into ciphertexts on the other kind, such that at the end, we can make computations with the two libraries and formats. 

Also, explain here the two kinds, or don't mention "This feature targets two main use cases.". Else, it's too enigmatic